docs/site/Crafting-LoopBack-4.md

```diff
@@ -473,7 +473,8 @@ There are several key pillars to make extensibility a reality for LoopBack 4.
 - [Context](Context.md), the IoC container to manage services
 - [Dependency injection](Dependency-injection.md) to facilitate composition
 - [Decorators](Decorators.md) to supply metadata using annotations
-- [Component](Component.md) as the packaging model to bundle extensions
+- [Component](Extending-LoopBack-4.md) as the packaging model to bundle
+  extensions
 
 Please check out [Extending LoopBack 4](Extending-LoopBack-4.md).
```

Comment:
Should we still keep this?

Reply:
I think we should still keep this bullet, because it belongs to `There are several key pillars to make extensibility a reality for LoopBack 4.` 
An alternative link for `Component` could be https://loopback.io/doc/en/lb4/Using-components.html. then it won't be duplicated what we have after the bullets.

what do you think?

Reply:
Yeah, I think that's better so we can avoid duplication.